src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -55,6 +55,7 @@ protected function configure(): void
                 new InputOption('show-aliases', null, InputOption::VALUE_NONE, 'Show aliases in overview'),
                 new InputOption('format', null, InputOption::VALUE_REQUIRED, \sprintf('The output format ("%s")', implode('", "', $this->getAvailableFormatOptions())), 'txt'),
                 new InputOption('raw', null, InputOption::VALUE_NONE, 'To output raw route(s)'),
+                new InputOption('sort', null, InputOption::VALUE_REQUIRED, 'Sort by column name'),
```

Comment:
Please add the list of possible values as last parameter, for bash completion.

```suggestion
                new InputOption('sort', null, InputOption::VALUE_REQUIRED, 'Sort by column name', null, ['name', 'method', 'scheme', 'host', 'path']),
```